pages/articles/[slug].tsx

```diff
@@ -198,41 +223,20 @@ const ArticlePage: NextPage = ({
 export const getServerSideProps = async (
   ctx: GetServerSidePropsContext<{ slug: string }>
 ) => {
-  const post = await prisma.post.findUnique({
-    where: {
-      slug: ctx.params?.slug,
-    },
-    select: {
-      id: true,
-      title: true,
-      body: true,
-      excerpt: true,
-      slug: true,
-      user: {
-        select: {
-          name: true,
-          image: true,
-          bio: true,
-          username: true,
-        },
-      },
-      tags: true,
-    },
-  });
+  console.log();
```

Comment:
console.log() ??

Reply:
Whoops I'll add a fix! I didn't get a notification about this